azure/build-image.md

```diff
@@ -88,11 +112,13 @@ export PKR_VAR_offer=0001-com-ubuntu-confidential-vm-jammy
 export PKR_VAR_sku=22_04-lts-cvm
 
 export AA_KBC="cc_kbc_az_snp_vtpm"
+export LIBC=gnu
```

Comment:
if we use glibc, do we still need to install the musl toolchain?

Reply:
Removed everything about musl.